fedimint-client/src/client.rs

```diff
@@ -792,6 +793,11 @@ impl Client {
         db: Database,
         num_responses_sender: watch::Sender<usize>,
     ) {
+        // Keep trying, initially somewhat aggressively, but after a while retry very
+        // slowly, because chances for response are getting lower and lower.
+        let mut backoff =
```

Comment:
More aggressive backoff might help when joining a federations and for whatever reason two connections to two peers failed randomly (network bleep or something). Does not seem very likely, but now it should behave better.